spec/model_rbi_formatter_spec.rb

```diff
@@ -37,8 +37,7 @@ def expect_match_file(content, file_path, save_to_file: false)
     formatter = ModelRbiFormatter.new(Wand, ['Wizard', 'Wand', 'SpellBook'])
     expect_match_file(
       formatter.generate_rbi,
-      'spec/test_data/models/expected_wand.rbi',
-      save_to_file: true,
+      'spec/test_data/models/expected_wand.rbi'
```

Comment:
It seemed strange to me that we were always overwriting the expectation file during tests—I couldn't get the test to fail at first without removing `save_to_file` here!

Reply:
Oh thanks, this is a bug. The `save_to_file` flag is to regenerate expected output file.

Reply:
Can you add a comma at the end of the line?